api/v1/usage_server.proto

```diff
@@ -73,7 +74,38 @@ message UsageDataPoint {
   int32 completion_tokens = 12;
 }
 
+message ListUsageDataRequest {
+  // start_time is the UNIX timestamp for the data start time (inclusive).
+  // If start_time is not provided, the default is the 24 hours before end_time.
+  int64 start_time = 1;
+  // end_time is the UNIX timestamp for the data end time (exclusive).
+  // If end_time is not provided, the default is the current time.
+  int64 end_time = 2;
+}
+
+message UsageDataByGroup {
+  string user_id = 1;
+  string api_key_id = 2;
+  string model_id = 3;
+
+  int64 total_requests = 4;
+  double avg_latency_ms = 5;
+  double avg_time_to_first_token_ms = 6;
+  int64 total_prompt_tokens = 7;
+  int64 total_completion_tokens = 8;
+}
+
+message ListUsageDataResponse {
+  repeated UsageDataByGroup usage_data = 1;
```

Comment:
maybe `user_datas` or `usage_data_groups`? Just prefer plural for a repeated field.